manifests/server.pp

```diff
@@ -158,18 +159,43 @@
           }
         }
       }
+    }
 
-      # Wrap the replset class
-      class { 'mongodb::replset':
-        sets => $replset_config_REAL
-      }
-      Anchor['mongodb::server::end'] -> Class['mongodb::replset']
+    # Wrap the replset class
+    class { 'mongodb::replset':
+      sets => $replset_config_REAL
```

Comment:
Ooo, I misread the indentation; it looked like the added `}` above was for the `if $replset {` on line 137, but it's for the `} else {` on 144. That's what I get for reading code on github's web diff.